doltcli/dolt.py

```diff
@@ -267,6 +267,8 @@ class Dolt(DoltT):
     """
 
     def __init__(self, repo_dir: str, print_output: Optional[bool] = None):
+        # allow ~ to be used in paths
+        repo_dir = os.path.expanduser(repo_dir)
```

Comment:
would you mind adding a test for sample usage in `tests/test_dolt.py`?

Reply:
Test added